### DISCUSSION TITLE
Consider using issuehunt

### DISCUSSION THREAD

**Discussion by Milo123459** [GitoxideLabs/gitoxide#278] [Ideas]
https://issuehunt.io is a way of adding bounties to issues, more or less incentivising people to contribute and solve issues. This could theoretically help the project reach more people and get more contributors / a larger following!

**Comment by Byron** [comment]
Thanks for the tip! I tried to sign up to take a look, and unfortunately it wants to have the keys to the castle right away :/.

<img width="727" alt="Screen Shot 2021-12-20 at 8 32 28 AM" src="https://user-images.githubusercontent.com/63622/146696665-0d5a15a7-ac97-4d6c-9677-86d4eb0531fb.png">

"I don't know you, so you can't act on my behalf" came to mind for a moment, and hopefully one day I find the courage.

**Reply by Milo123459** [reply]
Hold on.. thats a lot more perms then I remember. If it helps, things like BoostNOTE, yew.rs and refined-github all use this app.
